ESMF/Aerosol_GridComp/Aerosol_StateSpecs.rc

```diff
@@ -7,62 +7,64 @@ category: IMPORT
 #----------------------------------------------------------------------------------------
      NAME   | UNITS  | DIMS | VLOC| COND | LONG NAME
 #----------------------------------------------------------------------------------------
- DU_SRC     | 1      | xy   | N   |      | erod
- FRLAKE     | 1      | xy   | N   |      | fraction_of_lake
- WET1       | 1      | xy   | N   |      | surface_soil_wetness
- LWI        | 1      | xy   | N   |      | land-ocean-ice_mask
- U10M       | m s-1  | xy   | N   |      | 10-meter_eastward_wind
- V10M       | m s-1  | xy   | N   |      | 10-meter_northward_wind
- AREA       | m^2    | xy   | N   |      | agrid_cell_area
- USTAR      | m s-1  | xy   | N   |      | equivalent_neutral_10-meter_northward_wind
- CN_PRCP    |kg/m^2/s| xy   | N   |      | surface_conv._rain_flux_needed_by_land
- NCN_PRCP   |kg/m^2/s| xy   | N   |      | Non-convective precipitation
- ZPBL       | m      | xy   | N   |      | planetary_boundary_layer_height
- SH         | w m-2  | xy   | N   |      | sensible_heat_flux_from_turbulence
- Z0H        | m      | xy   | N   |      | surface_roughness_for_heat
+#inst_geop_interface | m2 s-2 | xyz | E | | inst_geop_interface
+#DU_SRC     | 1      | xy   | N   |      | erod
+#FRLAKE     | 1      | xy   | N   |      | fraction_of_lake
+#WET1       | 1      | xy   | N   |      | surface_soil_wetness
+#LWI        | 1      | xy   | N   |      | land-ocean-ice_mask
+#U10M       | m s-1  | xy   | N   |      | 10-meter_eastward_wind
+#V10M       | m s-1  | xy   | N   |      | 10-meter_northward_wind
+#AREA       | m^2    | xy   | N   |      | agrid_cell_area
+#USTAR      | m s-1  | xy   | N   |      | equivalent_neutral_10-meter_northward_wind
+#CN_PRCP    |kg/m^2/s| xy   | N   |      | surface_conv._rain_flux_needed_by_land
+#NCN_PRCP   |kg/m^2/s| xy   | N   |      | Non-convective precipitation
+#ZPBL       | m      | xy   | N   |      | planetary_boundary_layer_height
+#SH         | w m-2  | xy   | N   |      | sensible_heat_flux_from_turbulence
+#Z0H        | m      | xy   | N   |      | surface_roughness_for_heat
```

Comment:
Why are these commented?   Do we have a different way to get these quantities now?

Reply:
I'm just using this a template for now.

Reply:
These fields will eventually be needed, so they remain in the spec (but commented).